src/test/scala/sigmastate/lang/SigmaParserTest.scala

```diff
@@ -519,9 +519,9 @@ class SigmaParserTest extends PropSpec with PropertyChecks with Matchers with La
   }
 
   property("ZKProof") {
-    parse("ZKProof { condition }") shouldBe Apply(SigmaPredef.ZKProofSym, IndexedSeq(Ident("condition")))
+    parse("ZKProof { condition }") shouldBe Apply(Ident("ZKProof", SFunc(SSigmaProp, SBoolean)), IndexedSeq(Ident("condition")))
```

Comment:
@greenhat maybe refer to PredefinedFunc here?

Reply:
Fixed. Plus in other places where `Ident` was used.